cleanup_script.py

```diff
@@ -1,68 +1,88 @@
 import logging
-import os
 import datetime
+import os
 
 from cleanup_cli.app import executers
 from cleanup_cli.wrappers import env, connection
+from mc_automation_tools.parse import stringy
+
 
 """
-Script for deleting all ingestion tests data such as tile,db records, and mapproxy after running the tests set - 
-used to clean the resources and storage from unnecessary data 
+This script is used for cleaning our resources from unnecessary data from our tests ( nightly, manual tests.. ) 
+The data which will be removed : tiles from S3, DB records such as mapproxy, jobs & tasks and catalog records.
 """
-now = datetime.datetime.now()
-current_date = now.strftime("%Y-%m-%d")
 
-logger = logging.getLogger("cleanup_logger")
+def init_logger():
+    now = datetime.datetime.now()
+    current_date = now.strftime("%Y-%m-%d")
+    log_output_path = f"{os.environ.get('LOGS_FILE_PATH')}/{current_date}"
+
+    # init logger
+    logger = logging.getLogger("cleanup_logger")
+    file_handler = logging.FileHandler(log_output_path)
+    logger.addHandler(file_handler)
+    logger.setLevel(logging.INFO)
+
+    # create formatter and add it to the handlers
+    formatter = logging.Formatter('%(asctime)s - %(levelname)s - %(message)s')
+    file_handler.setFormatter(formatter)
+
+    return logger
+
+def logDataToClean(logger,data_to_clean):
+    logger.info(f"Found {len(data_to_clean)} layers to clean")
+    logger.info("\n" + stringy.pad_with_stars(
+        "The ID's of the layers found to be clean", length=138))
+    layersIDs = []
+    for layer in data_to_clean:
+        layersIDs.append(layer["identifier"])
+    logger.info(layersIDs)
 
-logger.setLevel(logging.INFO)
 
-log_file = f"{os.environ.get('LOGS_FILE_PATH')}/{current_date}"
```

Comment:
It's better to get the LOGS_FILE_PATH env var from config file and use it by import